dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFigcaption
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFigure(children = list(
+                htmlImg(src = 'https://brand.plot.ly/static/images/plotly-logo-01-stripe@2x.png'),
+                htmlFigcaption(children = 'Plotly Logo')))
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFigure
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFigure(children = list(
+                htmlImg(src = 'https://brand.plot.ly/static/images/plotly-logo-01-stripe@2x.png',
+                        width = '400',
+                        height = '150')
+                    )
+                  )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFont
+      dontrun: TRUE
+      code: |
+            # Starting with HTML 4, HTML does not convey styling information
+            # anymore (outside the <style> element or the style attribute of each
+            # element). CSS should be used for styling instead.
+    - name: htmlFooter
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFooter(list(
+                htmlH1('Dash'),
+                htmlLi('Pointer1'),
+                htmlLi('Pointer2')
+                    )
+                  )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlForm
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlForm(children=list(
+                htmlP(children=list('Username: ', dccInput(type='text', id='username', placeholder='username'))),
+                htmlP(children=list('Password: ', dccInput(type='password', id='password', placeholder='password'))),
+                htmlButton(children=list('Login'), type='submit', id='login_button')
+                    )
+                  )
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlFrame
+      dontrun: TRUE
+      code: |
+            # htmlFrame is now deprecated. htmlIFrame is recommended instead.
+    - name: htmlFrameset
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+            app <- Dash$new()
+
+            app$layout(htmlDiv(list(
+              htmlFrameset(children =
+              htmlIframe(width = "600px", height = "600px",
```

Comment:
really? I'd think `htmlFrameset` should just show a deprecation comment like `htmlFrame`

Reply:
Sounds good, will fix.

Reply:
fixed in 19551a7